service-loadbalancer/README.md

```diff
@@ -15,6 +15,35 @@ This module will not help with the latter
 
 ## Overview
 
+The image below roughly describes the service-loadbalancer alternative (compared with using NodePorts).
+
+```
+                          +----------------------+                     
+                          | haproxy load balancer|                     
+                          +---------+------------+                     
+                                    |                                  
+                +-------------------+---------------+                  
+                |                                   |                  
+   +------------+----------------+     +-----------+--------------------+
+   |kubelet role=loadbalancer (1)|     |kubelet role=loadbalancer (2)   |
+   +----+------------------------+     +--------------------------------+
+        |                                                       |
+        |                                                       |
+        | --tcpServices=postgresql:3333                         | --tcpServices=postgresql:3333  
```

Comment:
you started the loadbalancer with --tcpServices right? so why not put this up there in the haproxy box?